### PR TITLE
deployed Core Contracts on Citrea Chain Testnet

### PR DESCRIPTION
**Pull request by hoBabu1**
Added Network configuration of citrea testnet and deployed it .

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -11,6 +11,8 @@ const config: HardhatUserConfig = {
   etherscan: {
     apiKey: {
       lorenzo_testnet: "abc",
+      Citrea_Devnet:"def",
```
Comment (nikakhachi) [inline hardhat.config.ts]: Let's use lower case letters, so change it to: `citrea_devnet`

File: hardhat.config.ts [hunk 2]
```diff
@@ -22,6 +24,16 @@ const config: HardhatUserConfig = {
           apiURL: "https://scan-testnet.lorenzo-protocol.xyz/api/",
         },
       },
+      {
+        network: "Citrea_Devnet",
```
Comment (nikakhachi) [inline hardhat.config.ts]: `citrea_devnet`

File: hardhat.config.ts [hunk 3]
```diff
@@ -35,6 +47,10 @@ const config: HardhatUserConfig = {
         "7c5b27c4f043051e405d03469e3f9dfe5b65df74376dcaf70db003d63a976efc",
       ],
     },
+    Citrea_Devnet: {
```
Comment (nikakhachi) [inline hardhat.config.ts]: `citrea_devnet`

Comment (hoBabu1) [reply hardhat.config.ts]: I have made the changes.

Comment (nikakhachi) [reply hardhat.config.ts]: thanks, looks good

**Review by nikakhachi — APPROVED** [review]

**Pull request merged**